step.yml

```diff
@@ -43,6 +43,15 @@ inputs:
       description: |-
         The URL for the repository we are working with
       is_required: true
+  - git_ref: "$BITRISE_GIT_BRANCH"
```

Comment:
will `$BITRISE_GIT_BRANCH` be a valid value in case of a PR? In case of a PR there are target and destination branches also. Or this env is a shared common one?

Reply:
It seems to be set for PR builds (the source branch), but I am unsre if it is set in every case.
For example in the git-clone step, the checkout can be done even if none is specified:

```
func getCheckoutArg(commit, tag, branch string) string {
	switch {
	case commit != "":
		return commit
	case tag != "":
		return tag
	case branch != "":
		return branch
	default:
		return ""
	}
```